pkg/monitoring/config.go

```diff
@@ -214,7 +214,7 @@ func parseEnvVars(cfg *Config) {
 		cfg.Kafka.SecurityProtocol = value
 	}
 	if value, isPresent := os.LookupEnv("KAFKA_SASL_MECHANISM"); isPresent {
-		cfg.Kafka.SaslUsername = value
+		cfg.Kafka.SaslMechanism = value
```

Comment:
Wasn't this already merged? 
Yeah, it's here: https://github.com/smartcontractkit/chainlink-solana/commit/4e66fcb0db623a78a5d5a514c2a5ce31e689b3e9

Reply:
My bad, did a bad rebase, fixing now...